### DISCUSSION TITLE
WPF: Detect if Textbox Scrollbar is being dragged

### DISCUSSION THREAD

**Discussion by TonyValenti** [dotnet/runtime#48867] [Q&A]
How would I detect if a textbox's scrollbar is being dragged?  I have code that scrolls the textbox to the end but I don't want it to run if the user is scrolling the textbox themself.

**Comment by huoyaoyuan** [comment]
I believe there is no direct functionality for this.

My suggested solution is inherit TextBox, try to get the slider during OnApplyTemplate, then expose the state via a new property of your inherited control.
